tutorial.ipynb

```diff
@@ -490,20 +491,366 @@
     "\n",
     "keywords = Counter()\n",
     "for chunk in doc.noun_chunks:\n",
-    "    if nlp.vocab[chunk.lemma_].prob < - 8: # probablity value -8 is arbitraraly selected threshold\n",
+    "    if nlp.vocab[chunk.lemma_].prob < - 8: # probablity value -8 is arbitrarily selected threshold\n",
     "        keywords[chunk.lemma_] += 1\n",
     "\n",
     "keywords.most_common(20)"
    ]
   },
   {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": false
+   },
+   "source": [
+    "## Open task on the RAND Terrorism Dataset"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We have preprocessed the [RAND Terrorism Dataset](http://www.rand.org/nsrd/projects/terrorism-incidents.html) for this task. \n",
+    "\n",
+    "Can you find out the following using the code you have written?\n",
+    "\n",
+    "- Who are the persons mentioned in each article?\n",
+    "- What locations are mentioned in each article? Hint: a location just has a different label to a person\n",
+    "- From all of your entities, can you find out which named entities are terrorists from the syntactic relationships?\n",
```

Comment:
OK, just leave it there. We can decide about that tomorrow. Maybe @matichorvat comes up with nice syntactic solution.